frontend/src/components/Profile/BasicTabs.tsx

```diff
@@ -59,12 +69,46 @@ export default function BasicTabs() {
           <Tab label="subscriptions" {...a11yProps(2)} />
         </Tabs>
       </Box>
-      <TabPanel value={value} index={0}>
-        Item One
-      </TabPanel>
-      <TabPanel value={value} index={1}>
-        Item Two
-      </TabPanel>
+      {showEditPanel ? (
+        <TabPanel value={value} index={0}>
+          <form>
+            <Controller
+              control={control}
+              name="description"
+              render={({ field }) => (
+                <TextField
+                  placeholder="Please enter description"
+                  label="Description"
+                  fullWidth
+                  onChange={field.onChange}
+                  onBlur={field.onBlur}
+                  defaultValue={field.value}
+                  type="text"
+                />
+              )}
+            />
+            {/* <TextField
+              type="text"
+              label="Your Description"
+              fullWidth
+              sx={{ mt: '2vh' }}
+              onChange={setDescription}
+              value={description}
+            /> */}
```

Comment:
there are need for code for the future?

Reply:
i create frontend part write feedback on my pr